### PR TITLE
Em 65 -pop up if no summary letter exists

### PR DESCRIPTION
**Pull request by matrix-drorgo**

### CODE REVIEW

File: src/Components/Generic/patientTrackingTabs/setPatientDataWaitingForReleaseTableRows.js [hunk 1]
```diff
@@ -1,8 +1,20 @@
-import  { formatTime }  from 'Utils/Helpers/Datetime/formatDate';
+import { formatTime } from 'Utils/Helpers/Datetime/formatDate';
 import { goToEncounterSheet } from 'Utils/Helpers/goTo/goToEncounterSheet';
 import { getTableHeaders } from 'Components/Generic/patientTrackingTabs/tableHeaders';
 import { FHIR } from 'Utils/Services/FHIR';
 import { store } from 'index';
+import CustomizedPopup from '../../../Assets/Elements/CustomizedPopup';
```
Comment (matrix-amiel) [inline src/Components/Generic/patientTrackingTabs/setPatientDataWaitingForReleaseTableRows.js:6]: change to an absolute path

File: src/Utils/Helpers/showCustomizedPopUp.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import ReactDOM from 'react-dom';
+import CustomizedPopupStandAlone from '../../Assets/Elements/CustomizedPopupStandAlone';
```
Comment (matrix-amiel) [inline src/Utils/Helpers/showCustomizedPopUp.js:2]: change to an absolute path

File: src/Components/Generic/patientTrackingTabs/setPatientDataWaitingForReleaseTableRows.js [hunk 3]
```diff
@@ -1,8 +1,20 @@
...
+import React, { useState } from 'react';
+import ReactDOM from 'react-dom';
+import { text } from '@storybook/addon-knobs';
+import PopUpFormTemplates from '../PopupComponents/PopUpFormTemplates';
+import { createPortal } from 'react-dom';
+import CustomizedPopupStandAlone from '../../../Assets/Elements/CustomizedPopupStandAlone';
```
Comment (matrix-amiel) [inline src/Components/Generic/patientTrackingTabs/setPatientDataWaitingForReleaseTableRows.js:12]: change to an absolute path

File: src/Components/Generic/patientTrackingTabs/setPatientDataWaitingForReleaseTableRows.js [hunk 4]
```diff
@@ -1,8 +1,20 @@
...
+import { Typography } from '@material-ui/core';
+import {
+  destroyReactNoLetterPopUp,
+  showCustomizedPopUp,
+} from '../../../Utils/Helpers/showCustomizedPopUp';
```
Comment (matrix-amiel) [inline src/Components/Generic/patientTrackingTabs/setPatientDataWaitingForReleaseTableRows.js:17]: change to an absolute path

**Review by matrix-amiel — COMMENTED** [review]
Looks good!
just let's keep on an absolute path.
Thanks

File: src/Components/Generic/patientTrackingTabs/setPatientDataWaitingForReleaseTableRows.js (hunk 1 above)
Comment (matrix-drorgo) [reply src/Components/Generic/patientTrackingTabs/setPatientDataWaitingForReleaseTableRows.js:6]: not relevent - removed

File: src/Utils/Helpers/showCustomizedPopUp.js (hunk 2 above)
Comment (matrix-drorgo) [reply src/Utils/Helpers/showCustomizedPopUp.js:2]: not relevent - removed

File: src/Components/Generic/patientTrackingTabs/setPatientDataWaitingForReleaseTableRows.js (hunk 3 above)
Comment (matrix-drorgo) [reply src/Components/Generic/patientTrackingTabs/setPatientDataWaitingForReleaseTableRows.js:12]: not relevent - removed

File: src/Components/Generic/patientTrackingTabs/setPatientDataWaitingForReleaseTableRows.js (hunk 4 above)
Comment (matrix-drorgo) [reply src/Components/Generic/patientTrackingTabs/setPatientDataWaitingForReleaseTableRows.js:17]: fixed

**Pull request merged**